src/submission/validation-clinical/donor.ts

```diff
@@ -63,7 +85,7 @@ function checkTimeConflictWithSpecimens(
     return;
   }
   const specimenIdsWithTimeConflicts: string[] = [];
-  const donoSurvivalTime: number = Number(donorRecord[DonorFieldsEnum.survival_time]);
+  const donorSurvivalTime: number = Number(donorRecord[DonorFieldsEnum.survival_time]);
```

Comment:
wow, it's a spree. nice catch! 👍

Reply:
I do what I can ;)

Reply:
much appreciated! every little bit counts 💯